### DISCUSSION TITLE
Why I can't remove my own difficulty on map? (resolved but another problem is here)

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#20656] [Q&A] [answered]
New difficulty was created in osu!lazer and then I can't remove it... Only hide option is available. I can understand why is it disabled for ranked difficulties, but something strange with this case.
![osu_2022-10-09_10-56-53](https://user-images.githubusercontent.com/55319814/194745017-866df2dd-402e-48ab-8509-ef0c9aa7210e.jpg)

So I need to remove the whole mapset and then redownload it... Isn't it silly?

**Comment by okolenmi** [comment]
Okay. I find a way through Editor. But now this mapset is marked as "local". Looks like a bug. All difficulties except my are on their places.
It works for every mapset. Create your own difficulty and save it -> remove it -> mapset marked as local

![osu_2022-10-09_11-17-00](https://user-images.githubusercontent.com/55319814/194745723-2899c63a-c486-4b84-bab8-b08dfb76f38a.jpg)
![osu_2022-10-09_11-17-12](https://user-images.githubusercontent.com/55319814/194745727-df557e0e-86bd-4b66-8818-05200f21c6ce.jpg)

**Comment by Theighlin** [comment, marked as answer]
Duplicate of #20285
